README.md

```diff
@@ -41,8 +13,12 @@ Significance (#s of ppl dealing with these questions, examples of real-world res
 care?)
 
 🚧 Groundwork:
-Some history of the rise of equity compensation in the US and how it links to focus on
-shareholder value.
+Some
+[history](https://corpgov.law.harvard.edu/2014/10/02/what-has-happened-to-stock-options/) of
+the rise of equity compensation in the US and how it links to focus on shareholder value
+as well as
+[some counterpoint](https://www.wsj.com/articles/reviving-the-flagging-spirit-of-silicon-valley-1428706671)
+on how equity compensation has been key to Silicon Valley innovation.
 Any links about positive/negative/controversial aspects of this broad economic trend?
```

Comment:
Oddly I'm having trouble finding good links because Google only brings up "executive compensation controversy" when you try searching for equity controversy. Investopedia covers the basic pros/cons and has some links attached that I'll spend more time with later on: https://www.investopedia.com/articles/analyst/091202.asp

Reply:
I found a couple things that may be of interest here. One is this academic article about how IPOs can diminish innovation https://web.stanford.edu/~shaib/RP2126.pdf. Another is a lighter weight Fortune article about the decline of option granting in tech. http://fortune.com/2017/04/12/tech-stock-market/

Reply:
These are interesting too. One is a list of Case Studies by NCEO of companies that elected to offer equity compensation (though the page states it's copyrighted material and written permission is needed for use) https://www.hbs.edu/faculty/Publication%20Files/17-093_b4f6e873-ad56-4f49-a6b8-0d7b50cd07ab.pdf Another is a Harvard Business School paper about equality in salary and equity compensation which found that inequality aversion in equity is more severe than in salary because of the perceived scarcity of equity https://www.hbs.edu/faculty/Publication%20Files/17-093_b4f6e873-ad56-4f49-a6b8-0d7b50cd07ab.pdf